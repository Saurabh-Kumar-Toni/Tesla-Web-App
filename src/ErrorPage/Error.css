.error{
    padding: 100px 100px 100px 100px;
    justify-content: center;
    margin-top: 100px;
}