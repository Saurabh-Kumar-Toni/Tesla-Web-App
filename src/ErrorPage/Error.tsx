import React from 'react';
import '../ErrorPage/Error.css';

const Error: React.FC = () => {

    return (
        <div className="error">
            <h3>OOPS!! Looks Like this page doesn't exists</h3>

            <h5>Why don't you try to explore our Home Page</h5>
        </div>
    )
}

export default Error;