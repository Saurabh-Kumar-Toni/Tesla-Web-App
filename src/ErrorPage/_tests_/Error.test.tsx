import { render, screen } from '@testing-library/react';
import Error from '../Error'

test('check if the Error page is rendered properly or not', () => {
 
    const {container} = render(<Error />)
    expect(container.firstChild).toMatchSnapshot();
  });