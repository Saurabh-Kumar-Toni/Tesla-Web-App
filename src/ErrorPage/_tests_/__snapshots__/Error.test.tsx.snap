// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check if Form component is rendered properly or not 1`] = `
<div
  class="error"
>
  <h3>
    OOPS!! Looks Like this page doesn't exists
  </h3>
  <h5>
    Why don't you try to explore our Home Page
  </h5>
</div>
`;

exports[`check if the Error page is rendered properly or not 1`] = `
<div
  class="error"
>
  <h3>
    OOPS!! Looks Like this page doesn't exists
  </h3>
  <h5>
    Why don't you try to explore our Home Page
  </h5>
</div>
`;
